src/com/senlainc/warsaw/tyurin/Garage.java

```diff
@@ -0,0 +1,49 @@
+package com.senlainc.warsaw.tyurin;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Garage {
+
+    private Garage() {
+        craftsmen = new ArrayList<>();
+        orders = new ArrayList<>();
+    }
+
+    private static Garage INSTANCE;
+    private double space;
+    private List<Craftsman> craftsmen;
+    private List<Order> orders;
+
+    public static Garage getInstance() {
```

Comment:
Singleton nice :)
But you probably will change it in future, because of next tasks. So just keep it in mind for now